components/pierone.rst

```diff
@@ -63,3 +63,17 @@ Installation
 
 See the :ref:`STUPS Installation Guide section on Pier One <pierone-deploy>` for details about deploying Pier One into your AWS account.
 
+How to configure
```

Comment:
this should be below "Command Line Client" as the Pier One page is about the Docker registry itself